### ISSUE TITLE
how to interpret the png files from correctBias

### ISSUE THREAD

**Issue by tyler-hansen** [ReddyLab/CRADLE#33]
Hi again, 

Everything is working great so far, but I had a question on the png files that are produced from the correctBias function. I'm not really sure how to interpret them and I didn't see any thing about them in the paper or on the readme (I easily could be missing something). 

I assume they have to do with model fit, but I don't necessarily know what is good or bad. As an example, I attached one of the pngs to this. For context, we are using CRADLE to analyze a focused STARR-seq library, so only a small portion on the genome (~2%) is represented.

Thank you! 

![fit_input_rep1](https://user-images.githubusercontent.com/61889919/113319268-e0f8f200-92d6-11eb-864f-585a440233be.png)

**Comment by Young-Sook** [comment]
Hi Tyler,

Sorry that we missed the explanation about png files. As you expected, the png file are the model fit where a subset of training set was selected and plotted. The number on the right bottom in the png file is pearson correlation. 

When we analyzed whole genome data, we had better fit as below. 
![ctrl_rep2](https://user-images.githubusercontent.com/25181978/113332332-07c32280-92ef-11eb-84d0-c7353259e07d.png)

We've observed worse fit for small libraries compared to whole genome library. I think there could be two possible explanations. 
1. Less bias in small libraries compared to whole genome libraries, maybe largely due to less PCR reactions. 
2. There are distinct biases in small libraries that were not modeled in CRADLE. 


Also, I just want to point several things out here. 

If you are using BACs for STARR-seq, different coverage across different bacs will lead to weird peak calling. We have a script to normalize different coverage across BACs. We will upload this script sometime soon, but feel free to reach out to me if you need the script now (yk162@duke.edu). 

If you are using capture STARR-seq, the biases from capturing step could be very huge and those biases were not modeled in CRADLE.

Let me know if you have further questions. 

Thanks,
Young-Sook

**Comment by tyler-hansen** [comment]
Thank you for the prompt response. This is very helpful. I'll reach out if I have further questions.
